appveyor.yml

```diff
@@ -20,9 +20,9 @@ install:
     - set PATH=%PATH%;"%ANDROID_HOME%\tools\bin"
 
     - yes 2> nul | sdkmanager --licenses > nul
-    - sdkmanager "build-tools;26.0.2"
+    - sdkmanager "build-tools;28.0.3"
 
-    - choco install gradle --version 3.4.1
+    - choco install gradle --version 4.6.0
```

Comment:
Actually, this should install the minimum version of Gradle needed to generate the wrapper with our actually required version. At least that was true before this change.